src/js/components/TaskTable.js

```diff
@@ -50,6 +52,15 @@ class TaskTable extends React.Component {
         sortFunction
       },
       {
+        className: this.getHostColumnClassname,
```

Comment:
Yeah it can be cached if the list of hosts and ports are the exact same. How should I define that?